src/ert/config/parsing/config_schema_deprecations.py

```diff
@@ -177,4 +177,25 @@
         "for the Ensemble Smoother update algorithm. "
         "Please use ENKF_ALPHA and STD_CUTOFF keywords instead.",
     ),
+    DeprecationInfo(
+        keyword="QUEUE_OPTION",
+        message="The LSF_LOGIN_SHELL queue option has been removed and no longer has any "
+        "effect. It has been used in the past to force the bsub command to use a "
```

Comment:
It has not been removed yet, it will probably have an effect if LSF_SERVER is configured (via ert-configurations or not) and the c-driver is used. 

I think we want to say that we don't want user to use this, because it will actually disappear in the future (when the C-driver is deleted)